### ISSUE TITLE
RakeTask options for formatter don't remove default

### ISSUE THREAD

**Issue by tigris** [bbatsov/rubocop#2263]
If you set a formatter in the rake task like below, the "default" formatter isn't removed from the formatter set, and you end up with both formatters. As such there is no way to set a "default" formatter via rake.

```ruby
RuboCop::RakeTask.new(:rubocop) do |t|
  t.options = %w(--format json)
end
```

**Comment by jonas054** [comment]
There's a special attribute for rake tasks called `formatters` that you should use instead of `options`. See https://github.com/bbatsov/rubocop#rake-integration

**Comment by tigris** [comment]
You're right. This works as expected. I wonder if the above still sounds like a "bug". Completely up to you, happy to just close it if you want.

**Comment by jonas054** [comment]
I think we should issue a warning if `-f`/`--format` appears in `t.options`, so please leave this open.
